### DISCUSSION TITLE
Exporting  Data from Interrelated Tables to CSV from App

### DISCUSSION THREAD

**Discussion by vinnyWebDev** [Budibase/budibase#13812] [Help] [answered]
Hi everyone,

I have a Student table with various data fields such as name, age and nationality. There is a host_family column that links to another table. Additionally, we have arr_date and dep_date columns, which are derived from the Group table via a formula.

I need to export the Student table to a .csv file using a button action. However, this action currently does not export data that is related to other tables.

Is there a way to export all the data visible in the app's table, including the related data from other tables?

Thanks in advance for any help.

**Comment by mikesealey** [comment, marked as answer]
Hi @vinnyWebDev 

Unfortunately this functionality isn't available with Budibase itself, but as it's an open source project with a strong community, you may be able to find a [custom plugin](https://docs.budibase.com/docs/custom-plugin) or even write your own. [Find out how here](https://docs.budibase.com/docs/quickstart-plugins).

**Reply by vinnyWebDev** [reply]
Thank you, I'll investigate further.

**Reply by ajiitesh** [reply]
Hi @vinnyWebDev , were you able to find more information on this topic?
